include/NovelRT/Atom.h

```diff
@@ -55,54 +55,24 @@ namespace NovelRT
             return _value >= other._value;
         }
 
-        operator uintptr_t() noexcept
+        operator uintptr_t() const noexcept
         {
             return _value;
         }
 
         // TODO: These should be internal to NovelRT
 
-        static Atom getNextEventHandlerId() noexcept
-        {
-            static std::atomic_uintptr_t _nextEventHandlerId(0);
```

Comment:
These being in the header meant that `A.dll` and `B.exe` would each have their own storage for the static local here, resulting in them not having consistent world views of the "next id"